lib/puppet/pops/binder/bindings_model_meta.rb

```diff
@@ -94,7 +94,7 @@ class ProducerProducerDescriptor < ProducerDescriptor
   # @api public
   #
   class LookupProducerDescriptor < ProducerDescriptor
-    contains_one_uni 'type', Puppet::Pops::Types::PAnyType
+    has_attr 'type', Object
```

Comment:
While this is ok for now, an Object type cannot be serialized/deserialized. Eventually it needs to be fixed or it will be impossible to for a non Ruby model creator to produce an instance of this model. It does not matter at the moment because there are no such use-cases.

It is a technical debt though and deserves a comment like '# TODO: Use real type, not Object'